q2_amr/tests/card/test_utils.py

```diff
@@ -1,12 +1,14 @@
+import os
+import shutil
 import subprocess
-from unittest.mock import patch
+from unittest.mock import call, patch
 
 import pandas as pd
 from qiime2.plugin.testing import TestPluginBase
 from test_mags import TestAnnotateMagsCard
 
-from q2_amr.card.utils import create_count_table, load_preprocess_card_db, read_in_txt
-from q2_amr.types import CARDDatabaseFormat
+from q2_amr.card.utils import create_count_table, load_card_db, read_in_txt
+from q2_amr.types import CARDDatabaseDirectoryFormat, CARDKmerDatabaseDirectoryFormat
 
 
 class TestAnnotateReadsCARD(TestPluginBase):
```

Comment:
In other tests when you have test data that is uniquely used by a single test, you put that data inside of the function. I think this is good if the data is small, but now here you put it outside. Whatever your preference may be I think it best to stay consistent, e.g. either placing small datums inside the test function itself or outside. 

I believe it is good practice to put shared test data resources in a setUp method (when it's reasonably small otw it should go in its own directory). Conventions aside I think this is useful because sometimes a test might accidentally modify the test data and then a second test uses this modified version. Using a setup method like the one below ensures that every test is isolated and starts with fresh data to use (because before each test method is executed, the setUp method is called).

```python
def setUp(self):
        # Common data setup
        self.user_data = {'username': 'test_user', 'password': 'secret'}
```